zebra-network/src/peer/minimum_peer_version.rs

```diff
@@ -1,3 +1,5 @@
+//! Watches for chain tip height updates to determine the minimum support peer protocol version.
```

Comment:
Optional: I just realized something that might be a typo: should this be `supported`?

```suggestion
//! Watches for chain tip height updates to determine the minimum supported peer protocol version.
```

Reply:
I moved an existing module doc to avoid duplicates and inconsistencies.

It seems like a typo that's easy to fix.